BinaryEncoding.md

```diff
@@ -126,7 +129,7 @@ The type section declares all function signatures that will be used in the modul
 #### Type entry
 | Field | Type | Description |
 | ----- |  ----- | ----- |
-| form | `uint8` | `0x40`, indicating a function type |
+| form | `varuint7` | `0x40`, indicating a function type |
```

Comment:
True, that might make sense.  My hope is that we can use `varuint*` as the go-to default for everything and that only b/c of the extreme prevalence of opcodes (assuming the opcode-with-immediates table) *and* the consequent win of doubling the number of 1-byte opcodes do we make an exception for opcodes.